src/foam/build/DirCrawlModelDAO.js

```diff
@@ -84,7 +84,7 @@ foam.CLASS({
 
       context.foam.LIB = function(m) {
         if ( ! m.id ) {
-          m.id = 'lib.Lib' + Date.now()
+          m.id = m.name + '.' + m.methods[0].name;
```

Comment:
What libs have name but not id?  We should fix that.

Reply:
Just about all of them lol. We have stuff like:
```
foam.LIB({
  name: 'foam.Function',
  methods: [
    function argsStr() { ... },
  ]
})
foam.LIB({
  name: 'foam.Function',
  methods: [
    function args() { ... },
  ]
})
```

Reply:
Ohhh, I follow.